src/api/controllers/adminPermissionsController.js

```diff
@@ -62,7 +66,7 @@ export default class adminPermissions {
     const { email } = req.params;
     const { firstName, lastName } = req.body;
 
-    const result = await User.update({
+    await User.update({
```

Comment:
@Ramadhan0 why did you change this line?

Reply:
this result was not being used any where @m